exhaust.py

```diff
@@ -116,10 +116,16 @@ def main():
 
     args = parser.parse_args()
 
+    if args.verbose:
```

Comment:
Instead of having all these `if args.verbose` it may be better to make use of the python `logging` library.
With these we can set a logger that add logs based on a logging level (in this case the logging level would be `DEBUG`).

Reply:
Sorry it took me so long to get this done. I've implemented the logging. Is this how you would like it? Also, I put the logging level as INFO (forgetting that you had mentioned DEBUG as the logging level). Would you like me to change it to DEBUG?

Reply:
Hi, no problem.
Regarding the logging info I would say it would be better to keep them as debug